test/Kestrel.Transport.FunctionalTests/Http2/ChromeTests.cs

```diff
@@ -0,0 +1,107 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+#if NETCOREAPP2_2
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+using Microsoft.AspNetCore.Server.Kestrel.FunctionalTests;
+using Microsoft.AspNetCore.Testing;
+using Microsoft.AspNetCore.Testing.xunit;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Testing;
+using Xunit;
+
+namespace Interop.FunctionalTests
+{
+    [OSSkipCondition(OperatingSystems.MacOSX, SkipReason = "Missing SslStream ALPN support: https://github.com/dotnet/corefx/issues/30492")]
```

Comment:
These skip conditions are specific to the HTTP2 test, not chrome in general, move them there for now.

Reply:
Move what where? Moving the test itself to another HTTP/2 specific test class like HandshakeTests or ShutdownTests doesn't really make sense even though it would preserve the behaviour. I'm not sure if we want to move the attribute to the assembly level either since there are functional tests that need to run on all OSes. Do you mean to move these HTTP/2 tests to a separate project?

Reply:
Move it from the class to the test method.

Reply:
I suppose we could add http1 tests in the future. I'll move this test out of the Http2 folder as well.